applications/tari_validator_node/tests/cucumber.rs

```diff
@@ -77,25 +77,25 @@ use crate::utils::{
 
 #[derive(Debug, Default, cucumber::World)]
 pub struct TariWorld {
-    base_nodes: IndexMap<String, BaseNodeProcess>,
-    wallets: IndexMap<String, WalletProcess>,
-    validator_nodes: IndexMap<String, ValidatorNodeProcess>,
-    indexers: IndexMap<String, IndexerProcess>,
-    vn_seeds: IndexMap<String, ValidatorNodeProcess>,
-    miners: IndexMap<String, MinerProcess>,
-    templates: IndexMap<String, RegisteredTemplate>,
-    outputs: IndexMap<String, IndexMap<String, VersionedSubstateAddress>>,
-    http_server: Option<MockHttpServer>,
-    cli_data_dir: Option<String>,
-    current_scenario_name: Option<String>,
-    commitments: IndexMap<String, Vec<u8>>,
-    commitment_ownership_proofs: IndexMap<String, RistrettoComSig>,
-    rangeproofs: IndexMap<String, Vec<u8>>,
-    addresses: IndexMap<String, String>,
-    num_databases_saved: usize,
-    account_public_keys: IndexMap<String, (RistrettoSecretKey, PublicKey)>,
-    claim_public_keys: IndexMap<String, PublicKey>,
-    wallet_daemons: IndexMap<String, DanWalletDaemonProcess>,
+    pub base_nodes: IndexMap<String, BaseNodeProcess>,
```

Comment:
TariWorld is OPEN TO THE PUBLIC lol

Reply:
lol, yeah think that is why guys put all the steps in one file - because they use the fields on TariWorld. #callthecodejanitor :P